AmplifyPlugins/Predictions/AWSPredictionsPluginIntegrationTests/IdentifyBasicIntegrationTests.swift

```diff
@@ -219,20 +120,19 @@ class IdentifyBasicIntegrationTests: XCTestCase {
         let operation = Amplify.Predictions.identify(type: .detectEntities,
                                                      image: url,
                                                      options: PredictionsIdentifyRequest.Options()) { event in
-                                                        switch event {
-                                                        case .success:
-                                                            completeInvoked.fulfill()
-                                                        case .failure(let error):
-                                                            XCTFail("Failed with \(error)")
-                                                        }
+            switch event {
+            case .success:
+                completeInvoked.fulfill()
+            case .failure(let error):
+                XCTFail("Failed with \(error)")
+            }
```

Comment:
Are you using an app to update the spacing?  What was wrong with the spacing before?  Typically, we try to keep our changesets as small as possible.